py/dml/structure.py

```diff
@@ -806,9 +807,12 @@ def mksaved(spec, parent):
     if astinit:
         dml.globals.device.add_init_data(obj)
     try:
-        realtype(crep.node_storage_type(obj))
+        typ = realtype(crep.node_storage_type(obj))
     except DMLUnknownType as e:
         raise ETYPE(obj, e.type)
+    else:
+        serialize.map_dmltype_to_attrtype(site, typ)
```

Comment:
We want to eventually make all traits always alive, and then this will be removed. I suggest open a jira for this and sprinkle greppable comments `# should be removed when SIMICS-NNN is resolved` where applicable, so we have a chance to clean it up afterward.

Reply:
invalidated by 54ee113